lms/djangoapps/program_enrollments/api/v1/views.py

```diff
@@ -0,0 +1,249 @@
+# -*- coding: utf-8 -*-
+"""
+ProgramEnrollment Views
+"""
+from __future__ import unicode_literals
```

Comment:
Same thing here as in test_views.py -- Please organize these into the three blocks